demo.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function clean() {
+    echo Stopping Go Demo Containers
+    docker-compose -f docker-compose.demo.yml stop
+
+    echo Removing Go Demo Containers
+    docker-compose -f docker-compose.demo.yml rm
+}
+
+function rebuild() {
+    echo Re-Building Go Demo Environment
+    docker-compose -f docker-compose.demo.yml build --no-cache
```

Comment:
could you explain the need for `--no-cache`?

Reply:
This forces a rebuild of the containers since docker will not notice if the contrast_security.yaml is changed between builds. This invalidates the cache to ensure the SE demo environment includes the latest yaml changes.

Reply:
The `COPY` command makes docker look at a checksum for the target file(s) and should rebuild that layer when the config file changes. Does this differ from what you're seeing locally? I can try to test and see.

Reply:
I tested this locally and my config was not being invalidated and re-built

Reply:
Another important note here, is that even if my local files are updated without the cache rebuild flag it will NOT pull the latest rewriter or contrast-service which are requirements of this demo system for the SE team.